bin/test/main.ml

```diff
@@ -18,6 +18,8 @@ open Mdx
 
 let src = Logs.Src.create "cram.test"
 module Log = (val Logs.src_log src : Logs.LOG)
+module Result = Mdx.Migrate_ast.Result
```

Comment:
that's not the right way to support this -- you should use [Result](https://github.com/janestreet/result)